dali/imgcodec/decoders/nvjpeg/nvjpeg_helper.h

```diff
@@ -25,6 +25,11 @@
 #include "dali/core/cuda_error.h"
 #include "dali/core/format.h"
 
+#if WITH_DYNAMIC_NVJPEG_ENABLED
+  bool nvjpegIsSymbolAvailable(const char *name);
```

Comment:
you don't seem to be using this in this PR.

Reply:
Fixed